modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -281,7 +281,7 @@ function editFamilyInfoFields(\Database $db)
     $relationship     = isset($_POST[$relationshipType]) ?
         $_POST[$relationshipType] : null;
 
-    while ($siblingCandID != null ) {
+    if ($siblingCandID != null ) {
```

Comment:
This was an infinite loop, changing it to an `if` maintains the side-effect that would have happened inside the loop if it was ever entered.